docs/UserGuide.md

```diff
@@ -316,16 +316,10 @@ Notes about search keywords:
 
 * You must provide at least one keyword to search for.<br>
   e.g. entering just `find n/` alone is not a valid command as the keyword is empty.
-
-* Tags must only have one keyword.<br>
-  e.g. `find t/zoom math` is invalid. To search by multiple tags, you can do `find t/zoom t/math`.
```

Comment:
I'll probably reformat find to explain the cases more clearly. Can you revert this first, I'll change on my side, thanks.